<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>capture_main</class>
 <widget class="QWidget" name="capture_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scene Capture</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="setup_tab">
      <attribute name="title">
       <string>Setup</string>
      </attribute>
      <layout class="QVBoxLayout" name="setupTab_verticalLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="help_groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Help</string>
         </property>
         <layout class="QVBoxLayout" name="help_verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string>When this script is run for the first time, a set will be created containing the current selection. Objects can be added or removed from the set to change what will be captured. All keyable attributes of any transform or shape nodes will be captured.

An snapshot image will of the capture will be stored in the playblasts/sceneCapture/&lt;Scene&gt; subfolder of the current Maya project. This snapshot will by default use the first camera found in the set. This can be overridden from the options below. The image dimensions set in the Render Globals will be used.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="options_groupBox">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="snapCam_label">
            <property name="text">
             <string>Snapshot camera:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="snapCam_comboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="captures_tab">
      <attribute name="title">
       <string>Captures</string>
      </attribute>
      <layout class="QVBoxLayout" name="capturesTab_verticalLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="newCapture_GroupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>New Capture - give each capture a unique name</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="captureName_label">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="captureName_lineEdit">
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="capture_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Capture</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="captures_groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Captures</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>272</width>
               <height>16</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="captures_verticalLayout">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="render_tab">
      <attribute name="title">
       <string>Render</string>
      </attribute>
      <layout class="QVBoxLayout" name="renderTab_verticalLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="render_groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Create preview render of each checked set</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>W/H</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="width_spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>2048</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="height_spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>1536</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="path_lineEdit">
            <property name="text">
             <string>C:/</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QComboBox" name="filetype_comboBox">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>jpg</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>exr</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QPushButton" name="render_button">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>170</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>170</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>170</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>RENDER</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
